components/server/src/server.ts

```diff
@@ -283,6 +285,7 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
             res.send(this.config.version);
         });
         app.use(this.oauthController.oauthRouter);
+        app.use('/installation-admin', this.installationAdminController.apiRouter);
```

Comment:
@MrSimonEmms This is publicly available at `gitpod.io/api/installation-admin`. Either we use a custom express app at another port (see this.monitoringApp) - or use the DB as interface in the first place.

/cc @JanKoehnlein

Reply:
@geropl personally, creating an additional port just seems messy. How would you feel about an auto-generated HTTP Basic auth adding to the endpoint?

Reply:
> creating an additional port just seems messy

@MrSimonEmms  Definitely agree. But worse then an outward facing endpoint? Dunno. I'm just not super happy with this integration point in general.
But yesterdays discussions, again. Let's start out with this and look for alternative approaches! :+1: :pray: 

> How would you feel about an auto-generated HTTP Basic auth adding to the endpoint?

If the token/password is stable (does not change when upgrading a given installation) fine with me.